code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1438,7 +1438,7 @@ var/list/rank_prefix = list(\
 			if(client.eye == shadow)
 				reset_view(0)
 				return
-			if(istype(above, /turf/simulated/open))
+			if(istype(above, /turf/simulated/open) || istype(above, /turf/space))
```

Comment:
Instead of either of these, check above.is_hole

That var will be true on space tiles, openspace tiles, and in future on any other flooring which is wide open